### PR TITLE
[DEP-2810][PR]:Feedback form - textarea missing label

### PR DESCRIPTION
**Pull request by pratikrana06**
# DEP-2810

Link to AT PR : https://github.com/hmrc/digital-engagement-platform-journey-tests/pull/84

## Checklist PR Raiser
 - [x]  I've ensured code coverage has not decreased
 - [x]  I've dealt with any new compilation warnings
 - [x]  I've ensured the team's coding standards have been met (TBC)?
 - [ ]  If relevant, I've created corresponding app-config-XXX changes?
 - [ ]  If I've created new test data, I've ensured there is no real person's data (see https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open) 
 - [x]  I've considered the impact my changes have on the UI/Journey tests (please do not break them)
-  [ ]  If relevant, I've added links to associated PRs

## Checklist PR Reviewer
 - [ ]  I've checked to ensure all relevant unit tests have been written
 - [ ]  I've checked to ensure the team's coding standards have been met
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [ ]  If I merge I will ensure I use squash and merge

### CODE REVIEW

File: app/assets/javascripts/views/postChatSurvey/PostChatSurveyDigitalAssistant.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 const html = `
     <div id="postChatSurvey">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l" id="legend_give_feedback" tabindex="-1">
-          <h1 class="govuk-fieldset__heading">Give feedback</h1>
-        </legend>
+          <h2 class="govuk-fieldset__heading" id="legend_give_feedback" tabindex="-1">Give feedback</h2>
```
Comment (lmugnai) [inline app/assets/javascripts/views/postChatSurvey/PostChatSurveyDigitalAssistant.js:3]: Hi @pratikrana06 I think we can remove the class from this h2 as it's not within a fieldset. Can we remove it and see what it looks like?

File: app/assets/javascripts/views/postChatSurvey/PostChatSurveyWebchat.js [hunk 2]
```diff
@@ -1,8 +1,6 @@
 const html = `
     <div id="postChatSurvey">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l" id="legend_give_feedback" tabindex="-1">
-          <h1 class="govuk-fieldset__heading">Give feedback</h1>
-        </legend>
+          <h2 class="govuk-fieldset__heading" id="legend_give_feedback" tabindex="-1">Give feedback</h2>
```
Comment (lmugnai) [inline app/assets/javascripts/views/postChatSurvey/PostChatSurveyWebchat.js:3]: Again, can we remove the class as it's not in a fieldset?

File: app/assets/javascripts/views/postChatSurvey/PostChatSurveyDigitalAssistant.js (hunk 1 above)
Comment (pratikrana06) [reply app/assets/javascripts/views/postChatSurvey/PostChatSurveyDigitalAssistant.js]: > Hi @pratikrana06 I think we can remove the class from this h2 as it's not within a fieldset. Can we remove it and see what it looks like?

Done

File: app/assets/javascripts/views/postChatSurvey/PostChatSurveyWebchat.js (hunk 2 above)
Comment (pratikrana06) [reply app/assets/javascripts/views/postChatSurvey/PostChatSurveyWebchat.js]: > Again, can we remove the class as it's not in a fieldset?

Done

**Review by lmugnai — APPROVED** [review]

**Pull request merged**